frontend/src/Editor/editor.theme.scss

```diff
@@ -0,0 +1,45 @@
+.empty-box-cont{
```

Comment:
if it is specific to sample data source , add it to sample data source related css file

Reply:
No this is specific to editor only